styles/main.css

```diff
@@ -1,433 +1,651 @@
-    align-self: flex-start;
+  align-self: flex-start;
 }
 .gap-sm {
-    gap : 5px;
+  gap: 5px;
 }
 
 .gap-md {
-    gap: 10px;
+  gap: 10px;
 }
 
 .gap-lg {
-    gap: 20px;
+  gap: 20px;
 }
 
 .center-items {
-    align-items: center;
+  align-items: center;
+}
+
+.vertical-center {
+  vertical-align: middle;
 }
 
 .self-center {
-    align-items: center;
+  align-items: center;
+}
+
+li.end {
+  height: 50px !important;
+  position: fixed;
+  margin-bottom: 30px;
+  align-self: flex-end;
+  bottom: 0;
 }
 
 /**/
 
 .tweet-button {
-    color: white;
-    fill: white;
-    background-color: var(--blue);
-    width: 80%;
-    padding: 10px 0px;
-    box-sizing: content-box;
-    align-items: center;
-    margin: 0 auto;
-    justify-content: center
+  color: white;
+  fill: white;
+  background-color: var(--blue);
+  width: 80%;
+  padding: 10px 0px;
+  box-sizing: content-box;
+  align-items: center;
+  margin: 0 auto;
+  justify-content: center;
+}
+
+.tweet-button-small {
+  color: white;
+  fill: white;
+  background-color: var(--blue);
+  width: 45%;
+  height: 35px;
+  box-sizing: content-box;
+  margin: 0 auto;
+  align-self: flex-end;
+}
+
+.disabled {
+  opacity: 0.5;
 }
 
 .follow-button {
-    background: black;
-    color: white;
-    border: none;
-    height: min-content;
-    align-self: center;
- }
+  background: black;
+  color: white;
+  border: none;
+  height: min-content;
+  align-self: center;
+}
 /* -------------------------------- */
 .img-fit {
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-    max-width: 100%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  max-width: 100%;
 }
 
 /* AVATARS */
 
 .avatar {
-    height: 50px;
-    width: 50px;
-    position: relative;
-    overflow: hidden;
-    border-radius: 50%;
-    align-self: baseline;
-    justify-self: center;
+  height: 50px;
+  width: 50px;
+  position: relative;
+  overflow: hidden;
+  border-radius: 50%;
+  align-self: baseline;
+  justify-self: center;
 }
 
 /* -------------------------------- */
 
 .container {
-    width: 1250px;
-    margin: 0 auto;
-    display: grid;
-    grid-template-columns: 250px 1fr;
+  width: 1250px;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: min-content 1fr;
 }
 
 li {
-    list-style: none;
-    margin: 15px 0;
+  list-style: none;
+  margin: 20px 0;
 }
 ul {
-    margin: 0;
-    padding: 0;
+  margin: 0;
+  padding: 0;
 }
 
 svg {
-    height: 90%;
+  height: 90%;
+  cursor: pointer;
+}
+
+header {
+  position: sticky;
+  height: 100vh;
+  top: 0;
 }
+
 .nav li a {
-    height: 100%;
+  height: 100%;
+}
+.nav li {
+  height: 35px;
+  vertical-align: middle;
 }
-.nav li{
-    height: 35px;
-    vertical-align: middle;
-} 
 .nav {
-    position: sticky;
-    top: 25px;
+  position: sticky;
+  top: 25px;
+  width: 250px;
+  height: 100%;
 }
 
-.active h2{
-    font-weight: bold;
+.active h2 {
+  font-weight: bold;
 }
 
```

Comment:
Does this make the Home-icon blue by default.

Reply:
The logo is actually an SVG and reacts to the `fill` attribute in CSS, this is just for font-wheight